File: lib/java/src/main/java/uapi/internal/schema/GetAuthUApiJson.java
package uapi.internal.schema;

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.stream.Collectors;

public class GetAuthUApiJson {

    public static String getAuthUApiJson() {
        final var stream = Thread.currentThread().getContextClassLoader().getResourceAsStream("auth.uapi.json");
        return new BufferedReader(new InputStreamReader(stream)).lines().collect(Collectors.joining("\n"));
    }

}

File: tool/uapicodegen/uapicodegen/uapi/internal/schema/GetAuthUApiJson.py
import os


def get_auth_uapi_json() -> str:
    file_path = os.path.join(os.path.dirname(
        __file__), "..", "..", "auth.uapi.json")
    file_path = os.path.normpath(file_path)
    with open(file_path, "r") as stream:
        return "\n".join(stream.readlines())
